src/Compilers/CSharp/Test/Emit/Emit/EndToEndTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Roslyn.Test.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Microsoft.CodeAnalysis.Test.Utilities;
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Emit
+{
+    public class EndToEndTests: EmitMetadataTestBase
+    {
+        [WorkItem(16669, "https://github.com/dotnet/roslyn/issues/16669")]
+        [Fact]
+        public void OverflowOnFluentCall()
```

Comment:
Can we add a comment indicating we want to ensure that we don't regress the level of fluent calls allowed here.  Make it less of a surprise when developers hit this.